src/vs/base/browser/ui/hover/hover.css

```diff
@@ -138,3 +138,8 @@
 	margin-bottom: 4px;
 	display: inline-block;
 }
+
+.monaco-hover-content .action-container a {
+	-webkit-user-select: none;
```

Comment:
@hediet what does this do to accessibility (does it prevent tabbing to the actions)?

Reply:
Based on the docs, looks like this isn't an issue.